### DISCUSSION TITLE
Block Communication Across Namespaces

### DISCUSSION THREAD

**Discussion by zer0stars** [linkerd/linkerd2#8683] [Help]
Hello, 

I have a cluster with ~10 namespaces, some are infrastructure based(external secrets), or mon(grafana,prometheus).

The same cluster contains team specific namespaces(team1, team2). 

I am looking to create a fairly simple policy that leaves metrics scraping and general communication unchanged(scraping, web hooks, etc), while blocking any requests from team1's namespace to team2's namespace. Is there an example of this simple policy?

Thank you.

**Comment by wmorgan** [comment]
This should be basically what you want: https://buoyant.io/2021/12/14/locking-down-network-traffic-between-kubernetes-namespaces/
